--- src/main.cpp.orig   2013-04-28 14:41:58.000000000 +0200
+++ src/main.cpp        2013-04-29 19:48:33.462003913 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/options.h.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/options.h       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/options.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/options.cpp       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/mainwindowimpl.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/mainwindowimpl.cpp       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/mainwindowimpl.h.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/mainwindowimpl.h       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/dialogoptionsimpl.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/dialogoptionsimpl.cpp       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/dialogoptionsimpl.h.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/dialogoptionsimpl.h       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/figletmanager.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/figletmanager.cpp       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/figletmanager.h.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/figletmanager.h       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/figletfonts.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/figletfonts.cpp       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/figletfonts.h.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/figletfonts.h       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/dialoginfoimpl.cpp.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/dialoginfoimpl.cpp       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *

--- src/dialoginfoimpl.h.orig  2013-04-28 17:27:09.000000000 +0200
+++ src/dialoginfoimpl.h       2013-04-29 19:55:59.177977347 +0200
@@ -15,7 +15,7 @@
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
-* Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
 *
 * Contact e-mail: Faster <faster3ck@gmail.com>
 *
